nevergrad/optimization/callbacks.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import json
+import time
+import warnings
+import datetime
+from typing import Optional, Any, Union, List, Dict
+from pathlib import Path
+import numpy as np
+from . import base
+
+
+class OptimizationPrinter:
+    """Printer to register as callback in an optimizer, for printing
+    best point regularly.
+
+    Parameters
+    ----------
+    num_eval: int
+        max number of evaluation before performing another print
+    num_sec: float
+        max number of seconds before performing another print
+    """
+
+    def __init__(self, num_eval: int = 0, num_sec: float = 60) -> None:
+        self._num_eval = max(0, int(num_eval))
+        self._last_time: Optional[float] = None
+        self._num_sec = num_sec
+
+    def __call__(self, optimizer: base.Optimizer, *args: Any, **kwargs: Any) -> None:
+        if self._last_time is None:
+            self._last_time = time.time()
```

Comment:
Maybe I did not understand the logic, but from what I get, `self._last_time` is never reset to `time.time()` after printing the recommandation